### PR TITLE
upNext countdown timer continuous playback implementation

### PR DESCRIPTION
**Pull request by sveeravelli**

### CODE REVIEW

File: sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java [hunk 1]
```diff
@@ -232,12 +232,13 @@ private void handlePlayPause() {
...
 
   private void handleLearnMore() {
-    //implment learn more
+    //implement learn more
   }
   
   private void handleUpnextDismissed() {
     WritableMap body = Arguments.createMap();
-    body.putBoolean("upNextDismissed", _isUpNextDismiss);
+    _isUpNextDismiss=true;
```
Comment (lental) [inline sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java]: spaces before after the equals

File: sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java [hunk 2]
```diff
@@ -232,12 +232,13 @@ private void handlePlayPause() {
...
+    body.putBoolean("upNextDismissed",_isUpNextDismiss);
```
Comment (lental) [inline sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java]: space after comma

**Comment by lental** [comment]
overall really nice, dunno why you closed it

File: sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java [hunk 3]
```diff
@@ -64,7 +64,7 @@
   private DiscoveryOptions discoveryOptions;
 
   private boolean _isFullscreen = false;
-  private boolean _isUpNextDismiss = true;
+  private boolean _isUpNextDismiss = false;
```
Comment (lental) [inline sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java]: past tense - isUpNextDismissed

**Comment by sveeravelli** [comment]
Resolved conflicts and also adjustments made according to comments.

File: sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java [hunk 4]
```diff
@@ -288,6 +291,10 @@ public void update(Observable arg0, Object arg1) {
       bridgeTimeChangedNotification();
     } else if (arg1 == OoyalaPlayer.PLAY_COMPLETED_NOTIFICATION) {
       bridgePlayCompletedNotification();
+      if(!_isUpNextDismissed)
+      {
+        handleUpnextClick();
```
Comment (lental) [inline sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java]: sorry one more! capital N

**Comment by lental** [comment]
good stuff, i like it

**Pull request merged**